packages/@adobe/spectrum-css-temp/components/commons/focus-ring.css

```diff
@@ -45,3 +45,14 @@
     }
   }
 }
+
+@media (forced-colors: active) {
+  .spectrum-FocusRing,
+  .spectrum-FocusRing-ring,
+  .spectrum-FocusRing--quiet {
+    --spectrum-high-contrast-focus-ring-color: xvar(--spectrum-high-contrast-focus-ring-color, Highlight);
```

Comment:
why does this redefine itself? shouldn't it just be Highlight?

Reply:
Some components, like ActionButton or the StepList, define or override `--spectrum-high-contrast-focus-ring-color` to use ButtonText instead of Highlight. See https://github.com/search?q=repo%3Aadobe%2Freact-spectrum%20--spectrum-high-contrast-focus-ring-color&type=code. This allows the overrides to work, and adds `--spectrum-high-contrast-focus-ring-color` if it hasn't already been defined.